build.sbt

```diff
@@ -15,6 +15,10 @@ import scalapb.compiler.Version.scalapbVersion
 lazy val LanguageVersions = Seq(LatestScala212, LatestScala211)
 lazy val LanguageVersion = LanguageVersions.head
 
+lazy val V = new {
+  val diffutils = "1.3.0"
+}
```

Comment:
Let's copy/paste this version number instead. There's already some duplication of version numbers in this build definition, so I don't think that fixing just one instance of duplication is worth it. I think it would still make sense to come up with a strategy to deal with this, so we could open a ticket for this.